### PR TITLE
skip empty field values to be conditions of deletion

### PR DESCRIPTION
**Pull request by funlake**
fix : 删除时发现只设定某两列值，无法删除整行policy，必须所有列都设置好条件。

### CODE REVIEW

**Comment by funlake** [comment]
测试脚本有问题，注释掉mysql测试，单独跑pg的测试没问题的。
image

File: adapter.go [hunk 1]
```diff
@@ -101,7 +101,6 @@ func (a *Adapter) createDatabase() error {
 		return err
 	}
 	o = orm.NewOrm()
-
```
Comment (hsluoyz) [inline adapter.go:104]: Where is the typo you said?

File: adapter_test.go [hunk 2]
```diff
@@ -126,9 +126,10 @@ func TestAdapters(t *testing.T) {
...
-	testSaveLoad(t, "mysql", "root:@tcp(127.0.0.1:3306)/")
-	testSaveLoad(t, "postgres", "user=postgres host=127.0.0.1 port=5432 sslmode=disable")
+	testSaveLoad(t, "mysql", "root:123456@tcp(127.0.0.1:3306)/")
+	//time.Sleep(time.Millisecond * 200)
+	testSaveLoad(t, "postgres", "user=postgres host=172.16.10.113 port=5432 password=123456 sslmode=disable")
 
-	testAutoSave(t, "mysql", "root:@tcp(127.0.0.1:3306)/")
-	testAutoSave(t, "postgres", "user=postgres host=127.0.0.1 port=5432 sslmode=disable")
+	//testAutoSave(t, "mysql", "root:123456@tcp(127.0.0.1:3306)/")
+	//testAutoSave(t, "postgres", "user=postgres host=172.16.10.113 port=5432 password=123456 sslmode=disable")
```
Comment (hsluoyz) [inline adapter_test.go:134]: Just modify your connection string in your local working tree for test, do not commit it here.

**Review by hsluoyz — CHANGES_REQUESTED** [review]
Thanks for the PR. Please fix based on my suggestions.

File: adapter_test.go (hunk 2 above)
Comment (funlake) [reply adapter_test.go:134]: sorry,will fix them later

**Comment by hsluoyz** [comment]
@funlake hi, can we continue this PR?

**Comment by hsluoyz** [comment]
This PR seems deactivated here. But we should extract the valuable part behind it to resolve the underlying issue.

@closetool

**Comment by hsluoyz** [comment]
Invalid issue: https://github.com/casbin/beego-orm-adapter/issues/10#issuecomment-773817405

**Pull request closed without merging**